lib/tasks/export_copy_card_orders.rake

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+namespace :reports do
+  namespace :export do
+    desc "Run the weekly copy card orders export up to end of day yesterday, and upload it to S3."
+    task weekly_copy_card_orders: :environment do
+
+      # Run the report for one week up to end of day on the most recent Sunday
```

Comment:
The stakeholders *might* want this to run with the latest data  - e.g. cut-off at midnight on the Monday

Reply:
The stakeholder seems to want overnight on a Tuesday: https://eaflood.atlassian.net/browse/RUBY-1066?focusedCommentId=373717, "When we’d like the report".

Reply:
Right, `Date.today.prev_occurring(:sunday) + 1.day` My bad. You had that spot on